src/sequence/vector/mod.rs

```diff
@@ -679,5 +688,53 @@ impl<'a, T> DoubleEndedIterator for Iter<'a, T> {
 
 impl<'a, T> ExactSizeIterator for Iter<'a, T> {}
 
+#[cfg(feature = "serde")]
+impl<T> Serialize for Vector<T> where T: Serialize {
+    fn serialize<S: Serializer>(&self, serializer: S) -> Result<S::Ok, S::Error> {
+        let mut state = serializer.serialize_seq(Some(self.len()))?;
```

Comment:
There is a `collect_seq()` that does all this.

Reply:
Will do